homeassistant/components/camera/ring.py

```diff
@@ -42,11 +45,29 @@ def async_setup_platform(hass, config, async_add_devices, discovery_info=None):
     ring = hass.data[DATA_RING]
 
     cams = []
+    cams_no_plan = []
     for camera in ring.doorbells:
-        cams.append(RingCam(hass, camera, config))
+        if camera.has_subscription:
+            cams.append(RingCam(hass, camera, config))
+        else:
+            cams_no_plan.append(camera)
 
     for camera in ring.stickup_cams:
-        cams.append(RingCam(hass, camera, config))
+        if camera.has_subscription:
+            cams.append(RingCam(hass, camera, config))
+        else:
+            cams_no_plan.append(camera)
+
+    # show notification for all cameras without an active subscription
+    for camera in cams_no_plan:
+        err_msg = "A Ring Protect plan is required for {}.".format(camera.name)
+        _LOGGER.error(err_msg)
+        hass.components.persistent_notification.create(
+            'Error: {}<br />'
+            'You will need to restart hass after fixing.'
+            ''.format(err_msg),
+            title=NOTIFICATION_TITLE,
+            notification_id=NOTIFICATION_ID)
```

Comment:
You're possibly adding multiple persistent notifications. This ID should be unique or only the last one will be shown.
From the persistent-notification docs: `If notification_id is given, it will overwrite the notification if there already was a notification with that ID.`

Reply:
@NovapaX  good point. I'll update it. I was not aware of this detail. Thanks for the heads up!